### ISSUE TITLE
x2012.chartStyle.CTColorStyle needs an @XmlRootElement annotation

### ISSUE THREAD

**Issue by Amulya-sb** [plutext/docx4j#492]
I tried merge pptx enterprise edition trial version.
If Chart present in any slide - Merge is not working.

**Exception**
org.docx4j.openpackaging.exceptions.Docx4JException: Error marshalling JaxbXmlPart /ppt/charts/p1_colors1.xml
09:05:46.470 [main] ERROR org.docx4j.openpackaging.io3.Save - Error marshalling JaxbXmlPart /ppt/charts/p1_colors1.xml
org.docx4j.openpackaging.exceptions.Docx4JException: Error marshalling JaxbXmlPart /ppt/charts/p1_colors1.xml
	at org.docx4j.openpackaging.io3.stores.ZipPartStore.saveJaxbXmlPart(ZipPartStore.java:394)
	at org.docx4j.openpackaging.io3.Save.saveRawXmlPart(Save.java:289)
	at org.docx4j.openpackaging.io3.Save.savePart(Save.java:436)
	at org.docx4j.openpackaging.io3.Save.addPartsFromRelationships(Save.java:399)
	at org.docx4j.openpackaging.io3.Save.savePart(Save.java:456)
	at org.docx4j.openpackaging.io3.Save.addPartsFromRelationships(Save.java:399)
	at org.docx4j.openpackaging.io3.Save.savePart(Save.java:456)
	at org.docx4j.openpackaging.io3.Save.addPartsFromRelationships(Save.java:399)
	at org.docx4j.openpackaging.io3.Save.savePart(Save.java:456)
	at org.docx4j.openpackaging.io3.Save.addPartsFromRelationships(Save.java:399)
	at org.docx4j.openpackaging.io3.Save.save(Save.java:193)
	at org.docx4j.openpackaging.packages.OpcPackage.save(OpcPackage.java:788)

**Comment by plutext** [comment]
I tried merging a couple of pptx which contained waterfall charts (these have a chart colors part), and it worked for me.

Could you please provide further details to reproduce?  Feel free to email me the presentations you wish to merge.

**Comment by plutext** [comment]
From your complete stack trace:

```
com.sun.istack.SAXException2: unable to marshal type "org.docx4j.com.microsoft.schemas.office.drawing.x2012.chartStyle.CTColorStyle" as an element because it is missing an @XmlRootElement annotation
```

**Comment by plutext** [comment]
This issue caused by the introduction in docx4j 8.3.2 of dedicated parts for chart colors and style.
Fixed now by above commit.
Until a new release incorporating this fix, you could try reverting to docx4j 8.3.1.
